application/Common/HookLibrary/HookLibrary.vcxproj

```diff
@@ -135,6 +135,7 @@
       <ConformanceMode>true</ConformanceMode>
       <PrecompiledHeaderFile>
       </PrecompiledHeaderFile>
+      <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
```

Comment:
This setting, as opposed to `MultiThreadedDLL` (default) will statically link the VC++ libraries. If this causes problems (e. g. with windows defender), revert all occurences of this change to the default value and distribute the VC++ redistributables instead.